eZ/Publish/Core/Search/Common/FieldNameResolver.php

```diff
@@ -101,17 +92,17 @@ protected function getFieldMap()
                         continue;
                     }
 
-                    $this->fieldTypes[$contentType->identifier][$fieldDefinition->identifier] =
+                    $fieldTypes[$contentType->identifier][$fieldDefinition->identifier] =
                         $fieldDefinition->fieldType;
                 }
             }
         }
 
-        return $this->fieldTypes;
+        return $fieldTypes;
```

Comment:
Why removing `fieldTypes` property? Calling this method would have a non negligible cost AFAICS...